Sources/Errors.swift

```diff
@@ -10,6 +10,10 @@ import Foundation
 
 /// An error originating from ReactiveTask.
 public enum TaskError: Error, Equatable {
+
+	/// A shell task failed to launch
+	case shellTaskLaunchFailed(Task, reason: String?)
```

Comment:
This could probably be `launchFailed`. `shellTaskFailed` is meant to denote that the error comes from the shell task itself. In this case, the failure doesn't come from the shell task—it comes from the OS.